### PR TITLE
Parameterize the `run_batch.sh` script for GraphGym

### PR DESCRIPTION
**Pull request by jprorama**
Add bash defaults for the variables in the script.
Enables overriding the variable values on the commandline so the script
can easily reference different experiment, grid, etc values at runtime.

### CODE REVIEW

File: graphgym/run_batch.sh [hunk 1]
```diff
@@ -1,11 +1,11 @@
 #!/usr/bin/env bash
 
-CONFIG=example_node
-GRID=example
-REPEAT=3
-MAX_JOBS=8
-SLEEP=1
-MAIN=main
+CONFIG=${CONFIG:="example_node"}
+GRID=${GRID:="example"}
```
Comment (Padarn) [inline graphgym/run_batch.sh:4]:
```suggestion
GRID=${GRID:-example}
```
No need for the quality here as you are setting it anyway, the equality is used to do an assignment, see https://unix.stackexchange.com/questions/122845/using-a-b-for-variable-assignment-in-scripts
Also you don't need the quotes?

**Review by Padarn — COMMENTED** [review]
Looks sensible, but left comments on the style used

**Review by Padarn — APPROVED** [review]

**Comment by Padarn** [comment]
Can you add a note to the changelog?

**Review by rusty1s — APPROVED** [review]
Thanks!

**Pull request merged**